mu/config.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+import appdirs
+
+# The default directory for application data (i.e., configuration).
+DATA_DIR = appdirs.user_data_dir(appname="mu", appauthor="python")
+
+# The name of the default virtual environment used by Mu.
+VENV_NAME = "mu_venv"
+
+# The directory containing default virtual environment.
+VENV_DIR = os.path.join(DATA_DIR, VENV_NAME)

```

Comment:
Great, perhaps we should move more config options from logic.py to this file (not part of this PR), but maybe we should create an issue?